script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -13,22 +13,22 @@ Or is she coming straight towards me...?
 "Oh good, I've been looking for you, Tohno-kun.
 You saved me the trouble of having to visit your classroom."
 <Page6>
-"W-Well, that's great. Hello, Ciel-senpai."
+"W-Well, that's great. Good morning, Ciel-senpai."
```

Comment:
I changed this one because Ciel mentions how you wouldn't say こんにちは the second time you meet someone in the same day. I don't think "hello" really has the same connotation. Any better terms to use here?

Reply:
Hmm, "Good to see you"? A bit wordy. Morning just seems off since, well, it's lunchtime now so it's probably after noon.

Reply:
I'm fine with 'good to see you'